core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -169,6 +169,7 @@ public void put(String key, String value) {
 	 */
 	private synchronized Object setProperty(String key, String value, boolean local) {
 		if(local) {
+			//noinspection deprecation
```

Comment:
I don't understand this comment. Can you improve it?

Reply:
It's a comment for IntelliJ / Sonarlint that they should not highlight the use of a deprecated method.

Reply:
Ah, didn't know this syntax.. Usually `@SuppresWarnings("xxxx")` is used for that, afaik.

Reply:
This is what IntelliJ inserted for me. I think this is what you get when you want to ignore the warning for just a single statement, instead of a method or class.

Reply:
Ok fine.